osf/models/nodelog.py

```diff
@@ -143,6 +143,10 @@ class NodeLog(ObjectIDMixin, BaseModel):
 
     MIGRATED_QUICK_FILES = 'migrated_quickfiles'
 
+    RESOURCE_ADDED = 'resource_identifier_added'
+    RESOURCE_UPDATED = 'resource_identifier_udpated'
+    RESOURCE_REMOVED = 'resource_identifier_removed'
```

Comment:
I wonder if we should also update logActionsList.json and anonymousLogActionsList.json to handle the new log messages. It won't be immediately practical, but if we ever do end up updating the ember front end to display logs, we'll want entries in there or it'll probably be overlooked.

Reply:
I'll update the json files